1.8/administration/overlay-networks/ip-per-container.md

```diff
@@ -16,18 +16,16 @@ The overlay network feature is enabled by default in DC/OS. The default configur
 overlay. The `prefix` determines the size of the subnet (carved from the overlay subnet) allocated to each agent and thus defines the number of agents on which the overlay can run.
 
 In the default configuration above each overlay network is allocated a /8 subnet (in the “subnet” field), which is then divided into /26 container subnets to be used on each host that will be part of the network (in the “prefix” field) as shown:
 
-![Overlay network address space](/docs/1.8/administration/overlay-networks/img/overlay-network-address-space.png)
+![Overlay network address space](../img/overlay-network-address-space.png)
 
-With the default configuration each agent will be able to host a maximum of 2^5=32 Mesos containers and 32 docker containers. With this specific configuration, if a framework tries to launch more than 32 tasks on the Mesos containerzier or the Docker containerizer, it will result in a TASK_FAILURE. Please read the [limitations](#limitations) to understand more about this constraint in the system.
+The bits reserved for ContainerID (6 in this example) are then subdivided into two equal groups (of 5 bits in this example) that are used for Mesos containers and Docker containers respectively. With the default configuration, each agent will be able to host a maximum of 2^5=32 Mesos containers and 32 docker containers. With this specific configuration, if a service tries to launch more than 32 tasks on the Mesos containerizer or the Docker containerizer, it will receive a `TASK_FAILED`. Consult the [limitations](/docs/1.8/administration/overlay-networks/) section of the main Overlay Networks page to learn more about this constraint.
 
-These values can be configured to adapt to each installation’s needs. You can also add more overlay networks in addition to the default one, or modify/delete the existing overlay network. Currently, the addition/deletion of an overlay network is supported only at the time of installation. The next section describes how you can add more overlay networks to the existing default configuration.
+These values can be configured to adapt to each installation’s needs. You can also add more overlay networks in addition to the default one, or modify/delete the existing overlay network. Currently, the addition/deletion of an overlay network is supported only at install time. The next section describes how you can add more overlay networks to the existing default configuration.
```

Comment:
`You can modify the default overlay network configuration and add more overlay networks to fit your needs.`